Orange/base.py

```diff
@@ -383,6 +380,30 @@ def one_hot_probs(value):
                 probs[:, i, :] = one_hot(value[:, i])
             return probs
 
+        def extend_probabilities(probs):
+            """
+            Since SKLmodels and models with fit function does not guarantee
+            correct prediction dimensionality, extend dimensionality of
+            probabilities when it does not mach to the number of values in
+            domain.
```

Comment:
mach to the number of values in domain -> match the number of values in the domain